4_advanced_usage/30_technique_creation.txt

```diff
@@ -5,7 +5,94 @@ configuration and system administration needs. You can also create your own
 Techniques, to implement new functionality or configure new services. This
 paragraph will walk you through this process.
 
-==== Prequisites
+==== Use ncf-builder
+
+Easiest way to create your own Techniques is to use ncf-builder,
+a web interface to create and manage Technique based on ncf.
+
+Creating a technique in ncf-builder will generate a Technique for Rudder automatically
+that you can use as a Directive to apply on your Nodes.
+
+For more information about ncf and ncf-builder, you can visit: http://www.ncf.io/
+
+
+===== Organisation
```

Comment:
What should I put in "Understanding how Technique Editor works" ? global explanation about ncf-builder ?

And "Technique Editor integrations" is like the already present "Expalin technique editor integration" ?